### PR TITLE
Document the Fluent Plugin

### PR DESCRIPTION
**Pull request by roj1512**
- Got approved but mistakenly merged before translation in #228.

### CODE REVIEW

File: site/docs/.vuepress/config.ts [hunk 1]
```diff
@@ -210,6 +210,10 @@ export default defineUserConfig<DefaultThemeOptions>({
                     text: "Internationalization (i18n)",
                     link: "/plugins/i18n.md",
                   },
+                  {
+                    text: "Internationalization (fluent)",
+                    link: "/plugins/i18n-fluent.md",
```
Comment (WingLim) [inline site/docs/.vuepress/config.ts:215]: This name should not be `fluent.md`?

Comment (roj1512) [reply site/docs/.vuepress/config.ts:215]: Exactly, can you fix it, please?

Comment (WingLim) [inline site/docs/.vuepress/config.ts:215]:
```suggestion
                    link: "/plugins/fluent.md",
```

**Review by WingLim — CHANGES_REQUESTED** [review]

**Comment by roj1512** [comment]
No access to `slavafomin:plugins/fluent`. Closing and continuing in #252.

**Pull request closed without merging**